mteb/tasks/Classification/multilingual/AfriSentiLangClassification.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class AfriSentiLangClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="AfriSentiLangClassification",
+        description="AfriSentiLID is the largest LID classification dataset for African Languages.",
+        dataset={
+            "path": "HausaNLP/afrisenti-lid-data",
+            "revision": "f17cb5f3ec522ac604601fd09db9fd644ac66ca5",
+        },
+        reference="https://huggingface.co/datasets/HausaNLP/afrisenti-lid-data/",
+        type="Classification",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs={
```

Comment:
The only problem I see here, is that when you specify the language you want to benchmark on for this task, it will use the whole dataset with all other languages because there is no subset for each language.
`eval_langs` values are also used as parameters to select the language on which we'd liek to perform the evaluation.
it's similar to the discussion we had here: https://github.com/embeddings-benchmark/mteb/pull/480, I think it'll eb better if I open an issue to discuss these datasets that classify texts according to languages.

Reply:
@imenelydiaker I looked at the PR you pointed to but I am not sure if the resolution applies here? This is a Language Identification dataset unlike the one in the PR which is a toxicity classification dataset. So setting language to one majority language does not seem right to me. I was also following this tasks/Classification/multilingual/NordicLangClassification.py, which is similar problem to the one I added.